x-pack/plugins/alerting/common/rule_circuit_breaker_error_message.ts

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+
+const errorMessageHeader = 'Error validating circuit breaker';
+
+const getCreateRuleErrorSummary = (name: string) => {
+  return i18n.translate('xpack.alerting.ruleCircuitBreaker.error.createSummary', {
+    defaultMessage: `Rule '{name}' cannot be created. The maximum number of runs per minute would be exceeded.`,
+    values: {
+      name,
+    },
+  });
+};
+
+const getUpdateRuleErrorSummary = (name: string) => {
+  return i18n.translate('xpack.alerting.ruleCircuitBreaker.error.updateSummary', {
+    defaultMessage: `Rule '{name}' cannot be updated. The maximum number of runs per minute would be exceeded.`,
+    values: {
+      name,
+    },
+  });
+};
+
+const getEnableRuleErrorSummary = (name: string) => {
+  return i18n.translate('xpack.alerting.ruleCircuitBreaker.error.enableSummary', {
+    defaultMessage: `Rule '{name}' cannot be enabled. The maximum number of runs per minute would be exceeded.`,
+    values: {
+      name,
+    },
+  });
+};
+
+const getBulkEditRuleErrorSummary = () => {
+  return i18n.translate('xpack.alerting.ruleCircuitBreaker.error.bulkEditSummary', {
+    defaultMessage: `Rules cannot be bulk edited. The maximum number of runs per minute would be exceeded.`,
+  });
+};
+
+const getBulkEnableRuleErrorSummary = () => {
+  return i18n.translate('xpack.alerting.ruleCircuitBreaker.error.bulkEnableSummary', {
+    defaultMessage: `Rules cannot be bulk enabled. The maximum number of runs per minute would be exceeded.`,
+  });
+};
+
+const getRuleCircuitBreakerErrorDetail = ({
+  interval,
+  intervalAvailable,
+  rules,
+}: {
+  interval: number;
+  intervalAvailable: number;
+  rules: number;
+}) => {
+  return i18n.translate('xpack.alerting.ruleCircuitBreaker.error.ruleDetail', {
+    defaultMessage: `The {rules, plural, one {rule has} other {rules have}} {interval, plural, one {{interval} run} other {{interval} runs}} per minute; there {intervalAvailable, plural, one {is only {intervalAvailable} run} other {are only {intervalAvailable} runs}} per minute available. Before you can modify this rule, you must increase its check interval so that it runs less frequently. Alternatively, disable other rules or change their check intervals.`,
```

Comment:
> Before you can modify this rule, you must increase its check interval so that it runs less frequently. Alternatively, disable other rules or change their check intervals.

Is it feasible to change this message depending on whether it's a bulk or single rule scenario? Otherwise seeing "this rule" makes me question which rule it's referring to in the bulk case. I'm assuming in the bulk case you're trying to edit or enable multiple rules at the same time, so you can omit the "Before you can modify this rule..." and just have something like this:

> ... per minute available. Before you can modify these rules, you must disable other rules change their check intervals so they run less frequently.

Reply:
c5aea4e60be99ceeb43f5e3a8335a25abd394bcb